UPGRADE-7.0.md

```diff
@@ -121,6 +121,8 @@ FrameworkBundle
    $this->render(..., ['form' => $form]);
    ```
 
+ * Remove the integration of Doctrine annotations, use native attributes instead
```

Comment:
This was not supposed to be removed in Symfony 8?

Reply:
Nope.

Reply:
In https://github.com/symfony/symfony/pull/50888, you said:
> The plan is to make this the default and only valid option in Symfony 7 and remove that setting entirely in Symfony 8.

Reply:
Ohh sorry, just got it. The option cannot be switched on Symfony 7

Reply:
Yes, the setting is still there. You just cannot enable it.